community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -488,6 +488,12 @@
     @Description( "Log parameters for executed queries that took longer than the configured threshold." )
     public static final Setting<Boolean> log_queries_parameter_logging_enabled = setting( "dbms.logs.query.parameter_logging_enabled", BOOLEAN, TRUE );
 
+    @Description( "Log detailed time information for executed queries that took longer than the configured threshold." )
```

Comment:
Looking at those files, I don't see `dbms.logs.query.parameter_logging_enabled` in there either. Surely these settings should be on equal footing with that setting?

(Also: I'm sad that there is no sad-face emoji in github I can add as a reaction to the fact that we don't generate the config files anymore - do we at least have tests that verify that the commented out settings in there are settings that actually exist?)